packages/venia-ui/lib/components/CartPage/PriceAdjustments/__tests__/__snapshots__/priceAdjustments.spec.js.snap

```diff
@@ -32,7 +32,7 @@ exports[`it renders Venia price adjustments 1`] = `
         </span>
       </button>
       <div>
-        <ShippingMethods />
+        <() => Promise.resolve().then(() => _interopRequireWildcard(require('./ShippingMethods'))) />
```

Comment:
And I don't understand why the coverage comments do not show up here but they do in app.js...